CalendarApp/BookingRoom/Models/GoogleEvent/ToEventConverter.cs

```diff
@@ -0,0 +1,29 @@
+﻿using BookingRoom.Models.GoogleCalendar;
+using Google.Apis.Calendar.v3.Data;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Web;
+
+namespace BookingRoom.Models.GoogleEvent
+{
+    public static class ToEventConverter
+    {
+        private static Event _newEvent;
```

Comment:
Every time you are dealing with static resource you should consider multi threading and thread safety. So you should avoid this whenever possible